docker/Dockerfile

```diff
@@ -13,11 +13,11 @@ RUN apt-get update && \
     pip3 install --upgrade pip && \
     pip3 install setuptools==58.5.3
 
-RUN pip3 install torch -f https://download.pytorch.org/whl/torch_stable.html
+RUN pip3 install torch==1.13.1 torchvision==0.14.1 torchaudio==0.13.1 -f https://download.pytorch.org/whl/torch_stable.html
 RUN pip3 install --upgrade torch-scatter -f https://pytorch-geometric.com/whl
 
 # Install Ivy Upstream
-RUN git clone --progress --recurse-submodules https://github.com/unifyai/ivy && \
+RUN git clone --progress --recurse-submodules https://github.com/unifyai/ivy --depth 1 && \
```

Comment:
No need to add --depth 1 again in this pr, it will cause a merge conflict with master.